src/containers/Post/PostDetail.jsx

```diff
@@ -0,0 +1,29 @@
+import React, { Component } from 'react';
+
+import { Segment } from 'semantic-ui-react';
```

Comment:
Get rid of the line break above. Also when structuring imports. You should always provide imports of npm packages at the way top, then insert a new line break, and then provide the component, util functions, and etc. imports.